inc/components/components/site-menu/component.php

```diff
@@ -47,7 +50,7 @@
 				return $children;
 			}
 
-			$menu_items = wp_get_nav_menu_items( $config['menu_name'] );
+			$menu_items = wp_get_nav_menu_items( $config['menu_id'] );
```

Comment:
Suggestion: Lacking context here, but `wp_get_nav_menu_items` can also return `false`. Perhaps a change to something like `$menu_items = wp_get_nav_menu_items( $config['menu_id'] ) ?? [];` would guarantee `$menu_items` would always be an array.